contracts/multisig-prover/src/contract.rs

```diff
@@ -131,14 +133,14 @@ mod tests {
         )
     }
 
-    fn confirm_worker_set(test_case: &mut TestCaseConfig) -> Result<AppResponse, Error> {
+    fn confirm_worker_set(
+        test_case: &mut TestCaseConfig,
```

Comment:
would take a decent amount of refactoring to use that here, since the trait is coupled with the integration tests. The multisig prover tests need a lot of refactoring, and much of it should be moved to the integration tests package (there are still a lot of mocks here). I'd rather leave this as is for now, and tackle refactoring these tests as a separate task.